### DISCUSSION TITLE
Add (string) / escape + unescape

### DISCUSSION THREAD

**Discussion by wouterkroes** [rayepps/radash#340] [Ideas]
I would love to see [escape](https://lodash.com/docs/4.17.15#escape) + [unescape](https://lodash.com/docs/4.17.15#unescape), from Lodash, to be added to Radash.

For the implementation I was thinking of something like https://youmightnotneed.com/lodash#escape.

Is this something that would be appreciated?
If so, I would like to give it a go and make the pull request myself

Greetings

**Comment by aleclarson** [comment]
Hey! We have a similar proposal over at the [Radashi](https://github.com/radashi-org/radashi) fork.

See here: https://github.com/orgs/radashi-org/discussions/191
